sdf.md

```diff
@@ -1816,10 +1818,72 @@ The initial set of registrations is described in {{sdftype-r}}.
 Security Considerations {#seccons}
 =======================
 
-Some wider issues are discussed in {{-seccons}}.
-
-(Specifics: TBD.)
-
+Some wider security considerations applicable to Things are discussed
+in {{-seccons}}.
+{{Section 5 of -cddl}} gives an overview over security considerations
+that arise when formal description techniques are used to govern
+interoperability; analogs of these security considerations can apply
+to SDF.
+
+The security considerations of underlying building blocks such as
+those detailed in {{Section 10 of -utf8}} apply.
+SDF uses JSON as a representation language; {{-json}} for a number of
```

Comment:
Some punctation missing here? Or 
> For a number of cases {{-json}} indicates ...

Reply:
The sequence I used is valid but may indeed be surprising, so I changed this along the lines you suggested.

Reply:
... representation language. For a number of cases, {{-json}} indicates that ...